experiments/scripts/setup-sele.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+# Runs commands for "Instructions to setup a subject for test selection" section.
+
+# 1. Compile the new subject.
```

Comment:
If you are compiling the new subject in generate-sootTestOutput-auto.sh then you don't need to compile it here again. In fact, we should make another script that simply compiles the old and new subject. Doing so will make the purpose of our scripts much clearer. Please update the guidelines to reflect this as well.